test/Microsoft.ML.TestFramework/RemoteExecutor.cs

```diff
@@ -0,0 +1,201 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
Probably should have a comment, here or somewhere nearby, indicating where this code comes from - since someone might want to take an update from corefx one day.